packages/cli/src/cli/config/action.import.ts

```diff
@@ -330,15 +332,16 @@ export class CommandImport extends CommandLineAction {
     const vectorUpdate = [];
     const styleUpdate = [];
     for (const change of this.changes) {
-      if (change === 'ts_topographic') {
+      if (change.id.startsWith(ConfigPrefix.TileSet) && (change as ConfigTileSet).type === TileSetType.Vector) {
```

Comment:
I think this could be `mem.TileSet.is(change)` which would do the type casting for us.

Reply:
Good point, just updated.